### PR TITLE
Bump GUNC and add gunc/run test

### PR DESCRIPTION
**Pull request by dialvarezs**
This PR includes:
- Version bump in all GUNC modules
- A modification in the gunc/run module to accept multiple input files
- Add tests for gunc/run and gunc/mergecheckm

## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - For modules:
    - [x] `nf-core modules test <MODULE> --profile docker`
    - [x] `nf-core modules test <MODULE> --profile singularity`
    - [x] `nf-core modules test <MODULE> --profile conda`
  - For subworkflows:
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile docker`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile singularity`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile conda`

### CODE REVIEW

**Comment by dialvarezs** [comment]
@nf-core-bot run tests

**Comment by jfy133** [comment]
@dialvarezs this looks good but the module updates need to come via PRs into https://github.com/nf-core/modules and added to the pipline with `nf-core modules update <module>/<subcommand>`

**Comment by dialvarezs** [comment]
@jfy133 I don't get what you mean exactly. This is nf-core/modules, or am I doing something wrong?

**Comment by jfy133** [comment]
I mixed up my PRs, sorry @dialvarezs ! You're all good here :sweat_smile: (thoguht this was the mag PR...)

**Comment by dialvarezs** [comment]
I guess these tests are too much for GitHub runners 🤔

File: modules/nf-core/gunc/mergecheckm/tests/main.nf.test [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+                }
+            }
+
+            run("CHECKM_QA") {
+                script "../../../checkm/qa/main.nf"
+                process {
+                    """
+                    input[0] =  CHECKM_LINEAGEWF.out.checkm_output
+                                .join(CHECKM_LINEAGEWF.out.marker_file)
+                                .map { v -> v + [file('NO_FILE')] }
```
Comment (jfy133) [inline modules/nf-core/gunc/mergecheckm/tests/main.nf.test:123]: Heh, again please use more readable variable names ;) but not a blocker here

File: modules/nf-core/gunc/mergecheckm/tests/nextflow.config [hunk 2]
```diff
@@ -0,0 +1,5 @@
+process {
+  withName: CHECKM_QA {
+    ext.args = "--tab_table"
+  }
+}
```
Comment (jfy133) [inline modules/nf-core/gunc/mergecheckm/tests/nextflow.config:5]: There is nicer method of specifying these now with instructions merged in last week, so they are next to the test definition, try it out!

https://nf-co.re/docs/guidelines/components/modules#configuration-of-extargs-in-tests

File: modules/nf-core/gunc/run/meta.yml [hunk 3]
```diff
@@ -22,7 +22,7 @@ input:
         description: |
           Groovy Map containing sample information
           e.g. [ id:'test', single_end:false ]
-    - fasta:
+    - fasta_files:
         type: file
         description: FASTA file containing contig (bins)
```
Comment (jfy133) [inline modules/nf-core/gunc/run/meta.yml:27]:
```suggestion
        description: A list of FASTA files containing contig (bins)
```

**Review by jfy133 — APPROVED** [review]
LGTM!

File: modules/nf-core/gunc/mergecheckm/tests/nextflow.config (hunk 2 above)
Comment (dialvarezs) [reply modules/nf-core/gunc/mergecheckm/tests/nextflow.config]: I  tried it but setting the params inside `run()` doesn't seem to work (and it's not covered in the docs), so I left the args in `nextflow.config` again.

**Comment by dialvarezs** [comment]
@jfy133 Would it be possible to merge this despite the failing tests? The failures are because GitHub runners are running out of space while uncompressing the database, which seems unavoidable.

**Comment by jfy133** [comment]
Mmm it's not ideal... How big is the database?

**Comment by dialvarezs** [comment]
Around 25G uncompressed

**Comment by jfy133** [comment]
well poop.

I guess course of action:

1. See if the checkm2 Devs have their own very small test database we can 'borrow'
2. If not, remove tests (or comment out), only include stub test for GitHub auctions

**Pull request merged**